app/models/hud_reports/report_instance.rb

```diff
@@ -79,6 +90,19 @@ def failures
       report_cells.where.not(error_messages: nil).pluck(:question, :cell_name, :status, :error_messages)
     end
 
+    private def job_failed?
+      related_job.present? && related_job.failed?
+    end
+
+    def related_job
+      # See if we can find a related job (this is really overloading the jobs_for_class scope, but should work)
+      dj = Delayed::Job.jobs_for_class('RunReportJob').jobs_for_class(id.to_s)
```

Comment:
Does this condition work? If we're tyring to find either class, it seems like it should be
```suggestion
      dj = Delayed::Job.jobs_for_class(['RunReportJob', id.to_s])
```

Reply:
That's an intentional AND.  I'm mis-using the `jobs_for_class` method which really just does a `like` on the json `handler` column.  Basically, we want to ensure we get a row that is working on the right class AND has the right ID.